run.sh

```diff
@@ -83,7 +81,9 @@ if [[ $? -eq 0 ]]; then
     str_num=$(echo $status | sed -nr "s/.*на странице №([[:digit:]]{1,4})!.*/\1/p")
     echo "Задача $zad найдена на странице №$str_num!"
     
-    # open $sem $str_num
+    if [[ $open = true ]]; then
```

Comment:
Maybe better to operate with strings only in shell, like
```shell
open="true"
$open == $true"
```

And double equal operator

Reply:
Can't fix because pushing is blocked

Reply:
Fixed